firefox.go

```diff
@@ -51,8 +51,9 @@ func ReadFirefoxCookies(filename string) ([]*Cookie, error) {
 		cookie.Path = v
 
 		// Expires
-		v2, ok := rec.Values[7].(int32)
-		if !ok {
+		v2, ok1 := rec.Values[7].(int32)
+		_,  ok2 := rec.Values[7].(uint64)
```

Comment:
If uint64 values are going to be allowed in the column, I think the code below needs to handle them instead of throwing away the uint64 value.

Reply:
It works with the v2 variable from the int32 asertion. At least I get my cookies on my setup.